styles/lib/carnival/features/_tables.scss

```diff
@@ -0,0 +1,333 @@
+          padding-left: 0.7rem;
+        }
+        > td:not(:only-of-type) > strong {
+          word-break: break-word;
+        }
+        > td:not(:only-of-type):not(:last-child) {
+          border-left: none;
+          border-right: 0.025rem solid black; //TODO
+        }
+        > td {
+          border-bottom: 0.025rem solid black;
+          padding: 0.7rem;
+        }
+        > tr:last-of-type
+          > td:only-of-type {
+            border-right: none;
+          }
+    }
+    > table col:not(:last-child) {
+      border-right: 0.025rem solid black;
+    }
+  }
+  .os-table:not(.os-unstyled-container):after {
+    content: '';
+    width: 100%;
+    border-top-color: #0074BC;
+    border-top-width: 0.12rem;
+    border-top-style: solid;
+    display: block;
+    position: relative;
+    top: -0.1rem;
+  }
+
+  .os-table:not(.os-unstyled-container) .os-caption-container {
+    display: table-caption;
+    caption-side: bottom;
+    prince-caption-page: all;
+    font-size: 0.8333333333rem;
+    line-height: 1.2;
+    > .os-title-label {
+      font-family: Mulish, sans-serif;
+      font-weight: bold;
+      color: #C31427;
+      text-transform: uppercase;
+    }
+    > .os-number {
+      font-family: Mulish, sans-serif;
+      font-weight: bold;
+      color: #C31427;
+    }
+    > .os-title {
+      font-family: Mulish, sans-serif;
+      font-weight: bold;
+    }
+    > .os-caption {
+      font-family: IBM Plex Sans, sans-serif;
+    }
+  }
+
+}
+
+@mixin column-header-table(){
+  .os-table.os-column-header-container {
+    margin-bottom: 0.7rem;
+    display: table;
+    margin-left: auto;
+    margin-right: auto;
+    > table {
+      border-collapse: collapse;
+      > tbody
+        > tr
+          > td:not(:only-of-type) {
+            font-family: IBM Plex Sans, sans-serif;
+            border-left: 0.025rem solid black;
+            border-bottom: 0.025rem solid black;
+            padding-top: 0.7rem;
+            padding-bottom: 0.7rem;
+            padding-right: 0.7rem;
+            padding-left: 0.7rem;
+          }
+          > td:not(:only-of-type):not(:last-child) {
+            border-left: none;
+            border-right: 0.1rem solid black;
+            font-family: Mulish, sans-serif;
+            font-weight: 700;
+          }
+        > tr:first-of-type {
+          border-top-color: #0074BC;
+          border-top-width: 0.1rem;
+          border-top-style: solid;
+        }
+        > tr:last-of-type > td {
+          border-bottom-color: #0074BC;
+          border-bottom-width: 0.1rem;
+          border-bottom-style: solid;
+        }
+    }
+  }
+}
+
+@mixin vertically-tight-table(){
+  .os-table.os-vertically-tight-container > table.vertically-tight
+    > tr > th {
+      padding-top: 0.16rem;
+      padding-bottom: 0.16rem;
+    }
+    > tbody > tr > td:not(.regular-spacing):not(.pseudo-class-for-precedence) {
+      padding-top: 0.16rem;
+      padding-bottom: 0.16rem;
+    }
```

Comment:
this might be where some of your selectors are disappearing. it's missing the `{` on the first line (151)